incidents/form.py

```diff
@@ -0,0 +1,103 @@
+"""Streamlit app for incidents subscription form"""
+
+from os import environ as ENV
+from dotenv import load_dotenv
+
+import streamlit as st
+from boto3 import client
+
+from subscribe import on_submit
+
+
+OPERATORS = {'VT': 'Avanti West Coast',
+             'CC': 'c2c',
+             'CS': 'Caledonian Sleeper',
+             'CH': 'Chiltern Railways',
+             'XC': 'CrossCountry',
+             'EM': 'East Midlands Railway',
+             'XR': 'Elizabeth Line',
+             'ES': 'Eurostar',
+             'GC': 'Grand Central',
+             'LF': 'Grand Union Trains',
+             'GN': 'Great Nothern',
+             'GW': 'Great Western Railway',
+             'LE': 'Greater Anglia',
+             'HX': 'Heathrow Express',
+             'HT': 'Hull Trains',
+             'IL': 'Island Lines',
+             'LO': 'London Overground',
+             'LT': 'London Underground',
+             'LN': 'London Northwestern Railway',
+             'GR': 'LNER',
+             'LD': 'Lumo',
+             'ME': 'Merseyrail',
+             'NT': 'Northern Trains',
+             'SR': 'ScotRail',
+             'SE': 'Southeastern',
+             'SN': 'Southern',
+             'SW': 'South Western Railway',
+             'SP': 'Swanage Railway',
+             'SX': 'Stansted Express',
+             'TP': 'TransPennine Express',
+             'AW': 'Transport for Wales',
+             'TL': 'Thameslink',
+             'WR': 'West Coast Railways',
+             'LM': 'West Midlands Trains',
+             'GX': 'Gatwick Express',
+             'WM': 'West Midlands Railway'}
+
+
+if __name__ == "__main__":
+
+    st.set_page_config(page_title=None, page_icon='🔔', layout="wide",
+                       initial_sidebar_state="auto", menu_items=None)
+
+    load_dotenv()
+
+    sns = client('sns')
+    TOPIC = ENV["TOPIC_ARN"]
+
+    cont1, cont2, cont3 = st.columns((0.1, 1, 0.1))
+
+    with cont2:
+
+        with st.container(border=True, height=650):
+
+            st.markdown("<h2 style='text-align: center;'>Subscribe for alerts &#x1F514;</h2>",
+                        unsafe_allow_html=True)
+            st.markdown(
+                """<p style='text-align: center;
+                '>Enter your details below to receive
```

Comment:
Amazing work! Suggestion if you'd like to shorten this: 'Register below to receive live alerts and updates on service disruptions.'